addons/account/models/account_payment.py

```diff
@@ -314,6 +314,13 @@ def cancel(self):
                 move.unlink()
             rec.state = 'draft'
 
+    @api.model
+    def create(self, value):
```

Comment:
Normally when sending the email template, partner_to will be added as follower. This makes this code unnecessary. Could you check it is effectively the case please ?